openapi/listings/listings.yaml

```diff
@@ -261,6 +335,17 @@ components:
                   description: Does the website match the business’s website in their Vendasta Business Profile
                   type: boolean
             accuracyStatusCode:
-              description: Indicates if reporting is available. If available indicates if the listing was found. If the listing was found indicates if all the data is correct on the listing.
+              description: |-
+                Indicates if Vendasta's independent accuracy reporting is available, found and whether or not it is active.
+
+                unknown - Default error value, should never be used intentionally
+
+                accurate - Listing was independently verified by Vendasta to be accurate
+
+                inaccurate - Vendasta found at least one inaccuracy with the listing
+
+                notAvailable - No Vendasta reporting is available for this source
+
+                notFound - No listing was found for this source by Vendasta.
               type: string
               example: 'NOT_AVAILABLE, NOT_FOUND, INACCURATE, ACCURATE'
```

Comment:
These example values can be added as enum values instead of example values
```
              enum:
                - NOT_AVAILABLE
                - NOT_FOUND
                - INACCURATE
                - ACCURATE
```

Reply:
I removed this since it doesnt show up in the stoplight UI

Reply:
oops I committed the fix for this in https://github.com/vendasta/api-gateway-docs/pull/40